.github/workflows/golden_tests.yml

```
@@ -0,0 +1,101 @@
+
+jobs:
+  golden_tests:
+    name: Golden Tests
+    runs-on: ubuntu-latest
+    continue-on-error: true # Don't fail the build if golden tests fail
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Extract Flutter version
+        id: flutter-version
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC1009:info:1:94: The mentioned syntax error was in this simple command

(shellcheck)